addons/survey/controllers/user_input_session.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
```

Comment:
file naming : `session.py` is enouf I think.

Reply:
session.py makes me think of something related to "user session management".

Maybe `survey_session.py` ?